web-components/src/components/icons/Spinner.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import SvgIcon from '@mui/material/SvgIcon';
+import { keyframes } from '@mui/styled-engine';
+import { styled } from '@mui/material';
+
+interface IconProps {
+  className?: string;
+}
+
+const rotationKeyframe = keyframes`
+  from {
+    transform: rotate(359deg);
+  }
+  to {
+    transform: rotate(0deg);
+  }
+`;
+
+const Svg = styled(SvgIcon)({
+  animation: `${rotationKeyframe} 1.25s linear infinite`,
+});
+
+export const Spinner: React.FC<IconProps> = ({ className }) => (
+  <Svg
+    className={className}
+    xmlns="http://www.w3.org/2000/svg"
+    xmlnsXlink="http://www.w3.org/1999/xlink"
+    version="1.1"
+    x="0px"
+    y="0px"
+    viewBox="0 0 100 100"
+    enable-background="new 0 0 100 100"
```

Comment:
This relieves a console error
```suggestion
    enableBackground="new 0 0 100 100"
```